_faq/cep-and-its-importance.md

```diff
@@ -0,0 +1,25 @@
+---
+question: "What is a cEP and why is it so important to do pre-GSoC analysis and
+ design as early as possible, to be finalized in the cEP?"
+---
+A **cEP (coala Enhancement Proposal)** is proposed to the coala community to
```

Comment:
How about:
`A **cEP (coala Enhancement Proposal)** is a proposal to the coala community ...`? :sweat_smile:

Reply:
While I was preparing answer to the question I also thought that we should write something like you suggestbut then I thought mentioning `proposal` even after mentioning the full form of cEP won’t look nice. 
or even if we want to that word, writing `A cEP is a coala Enhancement Proposal that is proposed ...` 
What you think about it now ?

Reply:
This makes sense.
Looks good as it is :D

Reply:
Current one is also looking fine, defining the meaning that we want the user to undertand what actuall a cEP is !!

Reply:
Yea, that's what I said.
LGTM!

Reply:
Can you approve the changes ?